### ISSUE TITLE
question about data

### ISSUE THREAD

**Issue by wincj** [wtyhub/MuseNet#7]
Hello author, I have a question to ask.
Your job is to simulate various complex weather environments on the test set through data augmentation.
But I would like to ask if during the training process, you also used data augmentation, so the main reason why your model ultimately achieved good results in complex weather conditions is due to the data augmentation you applied to the training set? So if I use the same data augmentation method as you during the training process, will the model I use also achieve good results? If it is because of data augmentation to replace good results, I think this does not reflect the domain generalization ability, because the model only learns to perform retrieval under specific noise, and does not reflect generalization.
Have you conducted any relevant experiments to prove that the effectiveness of your model does not come from data augmentation in the training set? Thank you very much for your excellent work. I hope you can answer my questions. Thank you!

**Comment by wincj** [comment]
In addition, I have reviewed the results in your paper and found that under complex weather conditions (such as wind, snow, etc.), using other methods can also achieve retrieval results with only a slight decrease (compared to sunny days). Therefore, the main contribution of your method implementation is the improvement of retrieval performance, but this seems to have nothing to do with domain generalization under complex weather conditions, and is only due to the adoption of data augmentation on the training set.
The above is my personal understanding, and there may be some misunderstandings. I hope you can correct me. Thank you very much! Wish you a pleasant life
